src/components/OpenFileDialog/CustomSelectOption.js

```diff
@@ -1,28 +1,43 @@
+import i18n from '@dhis2/d2-i18n'
+import { MenuDivider, Tooltip } from '@dhis2/ui'
 import cx from 'classnames'
 import PropTypes from 'prop-types'
 import React from 'react'
 import styles from './styles/CustomSelectOption.style.js'
 
-export const CustomSelectOption = ({
+const CustomSelectOptionItem = ({
     value,
     label,
     icon,
+    insertDivider,
     onClick,
     active,
     disabled,
 }) => (
-    <div
-        onClick={(e) => onClick({}, e)}
-        data-value={value}
-        data-label={label}
-        className={cx({ active, disabled })}
-    >
-        {icon}
-        <span className="label">{label}</span>
-        <style jsx>{styles}</style>
-    </div>
+    <>
+        <div
+            onClick={(e) => onClick({}, e)}
+            data-value={value}
+            data-label={label}
+            className={cx({ active, disabled })}
+        >
+            {icon}
+            <span className={cx({ label: icon })}>{label}</span>
+            <style jsx>{styles}</style>
+        </div>
+        {insertDivider ? <MenuDivider dense /> : null}
+    </>
 )
 
+export const CustomSelectOption = (props) =>
+    props.disabled ? (
+        <Tooltip content={i18n.t('Not supported by this app yet')}>
```

Comment:
Just a note... Coupling the `disabled` prop to the case that something isn't supported but hinting that it will be in the future works for our (only) use-case of this, but also feels a bit limiting from Analytics perspective. (what if someone has a use-case where `disabled` is set for a different reason?). However extracting the message to a separate prop could be overcomplicating things as well...

Reply:
I guess we could have a different prop `unsupported` or something, which disables the option and adds this Tooltip, while the `disabled` one simply disables the option with no Tooltip.